client/src/app/pages/auth-page/auth-page.component.html

```diff
@@ -0,0 +1,41 @@
+<div class="container">
+
+    <mat-card >
+        <mat-card-subtitle *ngIf="isRegistered" class='caption'>{{ "app.auth-form.login" | translate }}</mat-card-subtitle>
+        <mat-card-subtitle  *ngIf="!isRegistered" class="caption">{{ "app.auth-form.register" | translate }}</mat-card-subtitle>
+        <form novalidate  class="form-control example-form" [formGroup]="userForm">
+          <mat-form-field class="example-full-width form-field">
+            <mat-label>{{ "app.auth-form.email-placeholder" | translate }}</mat-label>
+            <input matInput [formControl]="login" placeholder="Ex. pat@example.com">
+            <mat-error *ngIf="login.hasError('emailValidator') && !login.hasError('required')">
```

Comment:
!login.hasError('required') can be changed to dirty && touched. there are some examples in documentation how to display errors on fields